### ISSUE TITLE
Carthage build error

### ISSUE THREAD

**Issue by walkr** [Yalantis/Koloda#214]
running `carthage update --platform iOS` in my project gives me the following error

```shell
*** Skipped building Koloda due to the error:
Dependency "Koloda" has no shared framework schemes for any of the platforms: iOS
```

**Carthage:** 0.18
**Xcode:** 8
**OS X:** El Capitan
**Lang:** Swift 3

Any thoughts?

Thank you!

**Comment by shuvaev** [comment]
I have the same issue with the same environment. 
I only can add that I set "Use Legacy Swift Language Version" to NO in build settings.
I also tried to clone your sample project built with Carthage, and it doesn't build too. Build log stops on message "Check dependencies":
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

I also see a warning: "no umbrella header found for target 'Koloda', module map will not be generated"
